File: mycollab-web/src/main/java/com/esofthead/mycollab/module/crm/events/ActivityEvent.java
package com.esofthead.mycollab.module.crm.events;

import com.esofthead.mycollab.vaadin.events.ApplicationEvent;

@SuppressWarnings("serial")
public class ActivityEvent {

	public static class GotoCalendar extends ApplicationEvent {
		public GotoCalendar(Object source, Object data) {
			super(source, data);
		}
	}

	public static class GotoTodoList extends ApplicationEvent {
		public GotoTodoList(Object source, Object data) {
			super(source, data);
		}
	}

	public static class TaskAdd extends ApplicationEvent {
		public TaskAdd(Object source, Object data) {
			super(source, data);
		}
	}

	public static class TaskEdit extends ApplicationEvent {
		public TaskEdit(Object source, Object data) {
			super(source, data);
		}
	}

	public static class TaskRead extends ApplicationEvent {
		public TaskRead(Object source, Object data) {
			super(source, data);
		}
	}

	public static class MeetingAdd extends ApplicationEvent {
		public MeetingAdd(Object source, Object data) {
			super(source, data);
		}
	}

	public static class MeetingEdit extends ApplicationEvent {
		public MeetingEdit(Object source, Object data) {
			super(source, data);
		}
	}

	public static class MeetingRead extends ApplicationEvent {
		public MeetingRead(Object source, Object data) {
			super(source, data);
		}
	}

	public static class CallAdd extends ApplicationEvent {
		public CallAdd(Object source, Object data) {
			super(source, data);
		}
	}

	public static class CallEdit extends ApplicationEvent {
		public CallEdit(Object source, Object data) {
			super(source, data);
		}
	}

	public static class CallRead extends ApplicationEvent {
		public CallRead(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/scala/com/esofthead/mycollab/module/crm/events/ActivityEvent.scala
package com.esofthead.mycollab.module.crm.events

import com.esofthead.mycollab.eventmanager.ApplicationEvent

/**
  * @author MyCollab Ltd.
  * @since 5.0.3
  */
object ActivityEvent {

  class GotoCalendar(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoTodoList(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class TaskAdd(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class TaskEdit(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class TaskRead(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class MeetingAdd(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class MeetingEdit(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class MeetingRead(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class CallAdd(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class CallEdit(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class CallRead(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}
